sys/net/gnrc/network_layer/icmpv6/error/gnrc_icmpv6_error.c

```diff
@@ -192,16 +192,16 @@ static gnrc_pktsnip_t *_param_prob_build(uint8_t code, void *ptr,
     return pkt;
 }
 
-static void _send(gnrc_pktsnip_t *pkt, const gnrc_pktsnip_t *orig_pkt)
+static void _send(gnrc_pktsnip_t *pkt, const gnrc_pktsnip_t *orig_pkt
+                  gnrc_pktsnip_t *ipv6)
```

Comment:
Missing `,` between the last two arguments ;)

Reply:
I was sure I compiled that oO

Reply:
Addressed